### PR TITLE
Navbar added to book list view

### PR DESCRIPTION
**Pull request by renzowolox**
## Summary
- Navbar added on the book list view

## Screenshots
image
*View scrolled*
image

## Trello Card
https://trello.com/c/rRKunfj0/31-navbar#

### CODE REVIEW

File: maquetado/scss/list.scss [hunk 1]
```diff
@@ -1,25 +1,58 @@
...
-    display: flex;
-    flex-wrap: wrap;
-    justify-content: space-between;
-    margin: auto;
-    max-width: 800px;
-    width: 100%;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  margin: 75px auto;
```
Comment (Dorian30) [inline maquetado/scss/list.scss]: Avoid margin top and left for responsive reasons

File: maquetado/scss/list.scss [hunk 2]
```diff
@@ -1,25 +1,58 @@
...
 .book-coover-img {
-    width: 100%;
+  width: 100%;
 }
 
 .book-cover-title {
-    font-size: 12px;
-    font-weight: bold;
-    margin-top: 5px;
+  font-size: 12px;
```
Comment (leonardofrp5) [inline maquetado/scss/list.scss]: this font size you use a variable

File: maquetado/scss/list.scss [hunk 3]
```diff
@@ -1,25 +1,58 @@
...
+    background-image: url(images/LogoWolox.png);
+    background-repeat: no-repeat;
+    background-size: cover;
+    content: "";
+    height: 60px;
+    left: 20px;
+    position: absolute;
+    top: 0;
+    width: 180px;
+  }
```
[the remark this answers is not in the document]
Comment (renzowolox) [reply maquetado/scss/list.scss]: Image added :)

File: maquetado/scss/list.scss [hunk 4]
```diff
@@ -1,25 +1,58 @@
+.navbar {
+  background-color: $white;
+  border-top: 4px solid $dodger-blue;
+  box-shadow: 4px 4px 7px $gray-solid;
+  position: fixed;
+  top: 0;
+  width: 100%;
+}
+
+.navbar-content {
```
[the remark this answers is not in the document]
Comment (renzowolox) [reply maquetado/scss/list.scss]: Now, I'm using just one navigation tag

File: maquetado/scss/list.scss [hunk 5]
```diff
@@ -1,25 +1,58 @@
...
+  width: 100%;
 }
 
 .book-cover-title {
-    font-size: 12px;
-    font-weight: bold;
-    margin-top: 5px;
+  font-size: 12px;
+  font-weight: bold;
+  margin-top: 5px;
```
[the remark this answers is not in the document]
Comment (renzowolox) [reply maquetado/scss/list.scss]: Issue resolved

File: maquetado/scss/list.scss [hunk 6]
```diff
@@ -1,25 +1,58 @@
...
+  width: 100%;
 }
 
 .book-cover {
-    box-shadow: 4px 4px 7px grey;
-    flex-basis: 22%;
-    margin-bottom: 30px;
-    padding: 20px 25px;
+  box-shadow: 4px 4px 7px $gray-solid;
+  flex-basis: 22%;
```
[the remark this answers is not in the document]
Comment (renzowolox) [reply maquetado/scss/list.scss]: Issue resolved

File: maquetado/scss/list.scss (hunk 1 above)
Comment (renzowolox) [reply maquetado/scss/list.scss]: Margin top removed

File: maquetado/scss/list.scss (hunk 2 above)
Comment (renzowolox) [reply maquetado/scss/list.scss]: Issue solved

**Pull request merged**